### PR TITLE
Update Overview “portfolio-analysis”

### PR DESCRIPTION
**Pull request by onelittlebecca**
Automatically generated by Netlify CMS

### CODE REVIEW

File: _docs/overview/portfolio-analysis.md [hunk 1]
```diff
@@ -1,83 +1,93 @@
...
 
 Ideal applications follow as many of the [12-Factor App](https://12factor.net/) guidelines as possible. The more of the following questions you can answer with "yes", the better. Applications that follow these guidelines are easier to migrate between modern hosting environments; the guidelines aren't cloud.gov-specific except as noted!
 
 1. **Codebase**: Is the codebase tracked in version control?
-1. **Dependencies**: Are dependencies explicitly listed (such as package manager manifest)?
-1. **Configuration**: Can the configuration be retrieved from environment variables?
-1. **Backing services**: Does the application retry or fail gracefully if a service it connects to is unavailable?
-1. **Build, release, run**: Can you build the deployment artifacts outside of the deployment environment, for example in a continuous deployment system?
-1. **Processes**: Does the application keep its state in a backing service, rather than shared memory or filesystem?
-  - Applications that rely on a durable filesystem to share state between requests will be supported in FY19.
```
Comment (bengerman13) [inline _docs/overview/portfolio-analysis.md:67]: I _think_ this is the only content change. Is that right, @onelittlebecca ?

Comment (onelittlebecca) [reply _docs/overview/portfolio-analysis.md:67]: yes! Paired with @apburnes on it back in June!

**Review by bengerman13 — APPROVED** [review]

File: _docs/overview/portfolio-analysis.md (hunk 1 above)
Comment (bengerman13) [reply _docs/overview/portfolio-analysis.md:67]: awesome! I approved it, but it looks like there are some conflicts to resolve

**Comment by mheadd** [comment]
This proposed change was [made via another commit](https://github.com/cloud-gov/cg-site/commit/d54669ad5ccf4af60f151dd48b89f184d01bc9b8).

**Pull request closed without merging**